general_rules/Robots-OPL.tex

```diff
@@ -47,4 +50,11 @@ \subsubsection{Audio output plug}\label{rule:roobt_audio_out}
 		\item It is the responsibility of the teams to plug in the transmitter before a test, to check the sound system, and to hand over the transmitter to next team.
 		\item Do not rely on the sound system! For fail-safe operation and interacting with operators make sure that the sound system is not needed, e.g., by having additional speakers directly on the robot.
 \end{compactitem}
-\end{enumerate}
\ No newline at end of file
+\end{enumerate}
+
+
+
+
+\subsubsection{Appearance}
+\label{rule:robots_appearance}
```

Comment:
AFAIK no artificial markers of any kind are allowed in robots since... 2009 at least. Plus I must say that I haven't seen them in a while in @Home.

However, it is not my point to forbid something wasn't forbidden, but to clarify the rules. If we are unsure, we should ask @dirkholz, @komeisugiura, @swachsmu, and @iocchi to name some of the EC/Trustees who have been in the league longer than I.